@xen-orchestra/web/src/components/backups/jobs/panel/cards/BackupJobSettingsCard.vue

```diff
@@ -5,94 +5,88 @@
         </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="settings.concurrency !== undefined">
-        <template #key>{{ t('concurrency') }}</template>
-        <template #value>{{ settings.concurrency }}</template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="nbdConcurrency !== undefined">
-        <template #key>{{ t('nbd-concurrency') }}</template>
-        <template #value>{{ nbdConcurrency }}</template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="cbtDestroySnapshotData !== undefined">
-        <template #key>{{ t('cbt-destroy-snapshot-data') }}</template>
+      </UiLabelValue>
+      <UiLabelValue
+        v-if="settings.concurrency !== undefined"
+        :label="t('concurrency')"
+        :value="String(settings.concurrency)"
+      />
+      <UiLabelValue v-if="nbdConcurrency !== undefined" :label="t('nbd-concurrency')" :value="String(nbdConcurrency)" />
+      <UiLabelValue v-if="cbtDestroySnapshotData !== undefined" :label="t('cbt-destroy-snapshot-data')">
         <template #value>
           <VtsStatus :status="cbtDestroySnapshotData" />
         </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="maxExportRate !== undefined">
-        <template #key>{{ t('speed-limit') }}</template>
-        <template #value>{{ `${maxExportRate.value} ${maxExportRate.prefix}` }}</template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="settings.checkpointSnapshot !== undefined">
-        <template #key>{{ t('checkpoint-snapshot') }}</template>
+      </UiLabelValue>
+      <UiLabelValue
+        v-if="maxExportRate !== undefined"
+        :label="t('speed-limit')"
+        :value="`${maxExportRate.value} ${maxExportRate.prefix}`"
+        ellipsis
+      />
+      <UiLabelValue v-if="settings.checkpointSnapshot !== undefined" :label="t('checkpoint-snapshot')" ellipsis>
         <template #value>
           <VtsStatus :status="!!settings.checkpointSnapshot" />
         </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="settings.nRetriesVmBackupFailures !== undefined">
-        <template #key>{{ t('vm-backup-failure-number-of-retries') }}</template>
-        <template #value>{{ settings.nRetriesVmBackupFailures }}</template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="compression !== undefined">
-        <template #key>{{ t('compression') }}</template>
-        <template #value>{{ compression }}</template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue v-if="settings.offlineBackup !== undefined">
-        <template #key>{{ t('offline-backup') }}</template>
+      </UiLabelValue>
+      <UiLabelValue
+        v-if="settings.nRetriesVmBackupFailures !== undefined"
+        :label="t('vm-backup-failure-number-of-retries')"
+        :value="String(settings.nRetriesVmBackupFailures)"
+      />
+      <UiLabelValue v-if="compression !== undefined" :label="t('compression')" :value="compression" ellipsis />
```

Comment:
```suggestion
      <UiLabelValue v-if="compression !== undefined" :label="t('compression')" >
           <template #value>
               <VtsStatus :status="compression" />
          </template>
     </UiLabelValue>
```
Compression shouldn't use VtsStatus?

<img width="432" height="88" alt="image" src="https://github.com/user-attachments/assets/2c0e8823-1061-4662-aef6-556ba3833f60" />